db/migrate/20240411153910_create_accredited_individuals.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class CreateAccreditedIndividuals < ActiveRecord::Migration[7.1]
+  def change
+    create_table :accredited_individuals, id: :uuid do |t|
+      t.uuid :ogc_id, null: false
+      t.string :registration_number, null: false
+      t.string :poa_code, limit: 3
```

Comment:
```suggestion
      t.string :poa_code, limit: 3, index: true
```

There may be times we have to search just using poa_code. We won't need this one to be unique, it should match `full_name`.